src/AuditConfiguration.php

```diff
@@ -79,6 +81,16 @@ public function getTableName(ClassMetadataInfo $metadata)
         return $this->getTablePrefix().$tableName.$this->getTableSuffix();
     }
 
+    public function isDisabledForeignKeys(): bool
```

Comment:
Just for semantics, IMO it should be `areForeignKeysDisabled()`.

Reply:
I have made changes based on both comments.